integration_tests/snaps/simple-svn/snapcraft.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-package
+version: 0.1
+summary: svn integration test
+description: use svn
+confinement: strict
+
+parts:
+    svn:
+        plugin: make
+        source: .
+        source-type: svn
```

Comment:
@tsimonq2, I also think it makes us worry that you didn't run the test and discover the problem, which is kind of bad; or that you did run tests and this test should have run and failed but did not, which is really really bad. Please find out what the problem is, here.